### PR TITLE
lib: zigbee zcl scenes: Allow storing empty scenes [KRKNWK-14022]

### PR DESCRIPTION
**Pull request by sebastiandraus**
ChangeD zigbee scene library to allow storing scenes that do not have any fieldset set in them.

### CODE REVIEW

**Review by greg-fer — APPROVED** [review]
Approving, but please apply my comment.

**Review by robertstypa — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -658,7 +658,13 @@ Libraries for Zigbee
       *  ``zdo`` shell commands extended to allow binding to a group address.
       * Internal context manager structures.
 
-  * ref:`lib_zigbee_osif`:
+  * :ref:`lib_zigbee_zcl_scenes`:
+
+    * Updated:
+
+      * Zigbee ZCL scenes library allowed to store empty scenes.
```
[the remark this answers is not in the document]
Comment (sebastiandraus) [reply doc/nrf/releases/release-notes-changelog.rst]: Done

[the remark this answers is not in the document]
Comment (sebastiandraus) [reply doc/nrf/releases/release-notes-changelog.rst]: Done

**Comment by sebastiandraus** [comment]
Applied Greg suggestion

**Review by tomchy — APPROVED** [review]

**Pull request merged**